atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/SweepDelayTest.java

```diff
@@ -60,28 +60,32 @@ public void configurationAboveDefaultMaximumIsRespected() {
 
     @Test
     public void unableToAcquireShardReturnsMaxPause() {
+        delay.getNextPause(SUCCESS);
         assertThat(delay.getNextPause(SweepIterationResults.unableToAcquireShard()))
                 .isEqualTo(DEFAULT_MAX_PAUSE_MILLIS);
-        assertThat(metrics).hasValue(DEFAULT_MAX_PAUSE_MILLIS);
+        assertThat(metrics).hasValue(INITIAL_DELAY);
     }
```

Comment:
Do we have tests for the case where sweepDelayMetricsUpdater is updated to not INITIAL_DELAY?